src/commands/synthetics/README.md

```diff
@@ -185,43 +185,10 @@ All options under the `config` key are optional and allow overriding the configu
 - `variables`: (object) variables to replace in the test. This object should contain as keys the name of the variable to replace and as values the new value of the variable.
 
-<details>
-  <summary>(Deprecated) Usage of URL variables</summary>
-
-_The following has been deprecated since 0.18.1 and will be removed in a few versions. You should now use `startUrlSubstitutionRegex` or regular variable overrides._
-
-You can configure on which url your Browser or HTTP test starts by providing a `config.startUrl` to your test object and build your own starting url using any part of your test's original starting url and the following environment variables:
-
-| Environment variable | Description                  | Example                                                       |
-| -------------------- | ---------------------------- | ------------------------------------------------------------- |
-| `URL`                | Test's original starting url | `https://www.example.org:81/path/to/something?abc=123#target` |
-| `DOMAIN`             | Test's domain name           | `example.org`                                                 |
-| `HASH`               | Test's URL hash              | `#target`                                                     |
-| `HOST`               | Test's host                  | `www.example.org:81`                                          |
-| `HOSTNAME`           | Test's hostname              | `www.example.org`                                             |
-| `ORIGIN`             | Test's origin                | `https://www.example.org:81`                                  |
-| `PARAMS`             | Test's query parameters      | `?abc=123`                                                    |
-| `PATHNAME`           | Test's URl path              | `/path/to/something`                                          |
-| `PORT`               | Test's host port             | `81`                                                          |
-| `PROTOCOL`           | Test's protocol              | `https:`                                                      |
-| `SUBDOMAIN`          | Test's sub domain            | `www`                                                         |
-
-For instance, if your test's starting url is `https://www.example.org:81/path/to/something?abc=123#target`
-
-It can be written as :
-
-- `{{PROTOCOL}}//{{SUBDOMAIN}}.{{DOMAIN}}:{{PORT}}{{PATHNAME}}{{PARAMS}}{{HASH}}`
-- `{{PROTOCOL}}//{{HOST}}{{PATHNAME}}{{PARAMS}}{{HASH}}`
-- `{{URL}}`
-
-and so on...
-
-</details>
-
```

Comment:
👋  @DataDog/documentation @peculiarism Hey, we're removing the support for URL variables which has been deprecated for a few versions and reported with warnings  during the command execution.
I see we still document it on the [continuous testing configuration doc page](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration?tab=npm#starting-url), we should update it with `startUrlSubstitutionRegex` or usage of environment variables.

Reply:
I'll raise a PR to remove this from our docs, thanks for the heads up, @lefebvree!